WD-Firmware/comrv/api_inc/comrv_api.h

```diff
@@ -18,18 +18,89 @@
 * @file   comrv_api.h
 * @author Ronen Haen
 * @date   11.06.2019
-* @brief  The file defines the COM RV interfaces
+* @brief  The file defines the COM-RV interfaces
 */
 #ifndef __COMRV_TASK_API_H__
 #define __COMRV_TASK_API_H__
 
 /**
 * include files
 */
+#include "comrv_config.h"
 
 /**
 * definitions
 */
+// TODO: ronen - use psp defines
+#define D_COMRV_INLINE     __attribute__((inline))
+#define D_COMRV_NO_INLINE  __attribute__((noinline))
+#define _OVERLAY_          __attribute__((overlaycall)) D_COMRV_NO_INLINE
+
+/* make sure eviction algorithm defined */
+#ifndef D_COMRV_EVICTION_LRU
+ #ifndef D_COMRV_EVICTION_LFU
+  #ifndef D_COMRV_EVICTION_MIX_LRU_LFU
+   /* default eviction algorithm */
+   #define D_COMRV_EVICTION_LRU
+  #endif /* D_COMRV_EVICTION_MIX_LRU_LFU */
+ #endif /* D_COMRV_EVICTION_LFU */
+#endif /* D_COMRV_EVICTION_LRU */
+
+/* if no profile was set */
+#if D_COMRV_PROFILE==0
```

Comment:
double check if define 0 equals is defined

Reply:
done in 29e16d5